src/main/java/seedu/address/logic/commands/AddPatientCommand.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK_DESCRIPTION;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a patient to the patient list.
+ */
+public class AddPatientCommand extends Command {
+
+    public static final String COMMAND_WORD = "addPatient";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a patient to the patient list. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "PHONE "
+            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_TASK_DESCRIPTION + "TASK DESCRIPTION ";
+
```

Comment:
Was the decision to leave out the `PREFIX_TAG` intentional? I think we are keeping the tags (which was why we decided on `d/` for addition of tasks in the first place).